micro-rdk/src/common/conn/server.rs

```diff
@@ -411,19 +426,14 @@ where
             }
         }
     }
-    async fn serve_http2<U>(
-        &self,
-        mut c: U,
+    async fn serve_http2(
+        connection: T,
+        exec: Executor,
         robot: Arc<Mutex<LocalRobot>>,
-    ) -> Result<(), ServerError>
-    where
-        U: Http2Connector<Stream = T>,
-    {
+    ) -> Result<(), ServerError> {
         let srv = GrpcServer::new(robot.clone(), GrpcBody::new());
-        let connection = c.accept().await.map_err(|e| ServerError::Other(e.into()))?;
-
         Box::new(
-            http2::Builder::new(self.exec.clone())
+            http2::Builder::new(exec)
```

Comment:
⬇️ I think we may need to raise `max_concurrent_streams` to allow inbound gRPC/HTTP2 to function. At least with a python client, I see it attempt to open additional streams after some time connected, even when making a request every second, and opening that additional stream fails if this is set to one.

@npmenard - How costly would it be to set this to `2`, which seems sufficient to let a connected python client not fail after a short time.

Reply:
I've made the change and went to two streams permitted. The Python SDK, for one, has a periodic task which calls `RobotService/ResourceNames` to validate connection. Without the option for a second stream, this will fail and cause the python client to disconnect.

Reply:
Yes that tracks, we limited it to one for the ESP32-WROOM which we dropped